ci-operator/jobs/openshift/odo/openshift-odo-master-presubmits.yaml

```diff
@@ -93,6 +93,7 @@ presubmits:
         - --org=openshift
         - --repo=odo
         - --resolver-address=http://ci-operator-configresolver-ci.svc.ci.openshift.org
+        - --secret-dir=/tmp/secret
```

Comment:
Were these lines added by config -> jobs generator scripts or did you have to add them manually?
If you added these manually, it might be a good idea to add comments around these saying `do not remove - if removed please add this back`

Reply:
> Were these lines added by config -> jobs generator scripts or did you have to add them manually?
As far as I know these lines are added manually. @amitkrout Correct me if I am wrong.

> it might be a good idea to add comments around these saying do not remove - if removed please add this back
Coming to the comment +1 @mohammedzee1000 yes It is a good idea to add comment around. But IMO it would look better if we say comment as ```odo secret volume mount parameters``` 

Anyway we can add whatever suits perfect :-)

Reply:
I am ok with anything that tells the user why its there and more importantly not to remove it. If it is generated from config, then of course, this is not nessasary

Reply:
> Were these lines added by config -> jobs generator scripts or did you have to add them manually?

@mohammedzee1000 This is being added through the job generator. 

> As far as I know these lines are added manually. @amitkrout Correct me if I am wrong.

@prietyc123 since we last discussed steps has been changed. It was true that we are adding it manually, but for now we are not adding manual lines in the odo jobs file. See the reference - https://github.com/openshift/release/pull/6764#issuecomment-590918025